www/book.css

```diff
@@ -213,6 +212,28 @@ code .im { color: #a0a; }
 code .co { color: #C4A484; }
 }
 
+/* Tagging code blocks with where they go */
```

Comment:
I tested this PR locally and it doesn't seem to work? Also, some code blocks look broken, like the attached example.
![Screenshot 2024-04-12 4 36 29 PM](https://github.com/browserengineering/book/assets/3453258/32830f19-04c2-4037-a90d-9fb79cf55525)